app/src/main/java/com/keylesspalace/tusky/BaseActivity.java

```diff
@@ -92,11 +91,6 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         requesters = new HashMap<>();
     }
 
-    @Override
-    protected void attachBaseContext(Context base) {
```

Comment:
why do we not need it anymore?

Reply:
Because that was our custom hack, now that functionality is inside AndroidX and `AppCompatDelegate.setApplicationLocales` does everything for us